### PR TITLE
add doc requirement

### PR DESCRIPTION
**Pull request by jgrss**
## What is this PR changing?
* Adds all file geometries to `DataArray` mosaic
* Fixes import error
* Adds `pickleshare` requirement for Sphinx doc builds

### CODE REVIEW

File: src/geowombat/backends/xarray_.py [hunk 1]
```diff
@@ -452,7 +452,11 @@ def mosaic(
         filenames[0], nodata=ref_kwargs['nodata'], **kwargs
     ) as src_:
         attrs = src_.attrs.copy()
-        geometries = [src_.gw.geometry]
+
+    geometries = []
+    for fn in filenames:
+        with open_rasterio(fn, nodata=ref_kwargs['nodata'], **kwargs) as src_:
+            geometries.append(src_.gw.geometry)
```
Comment (jgrss) [inline src/geowombat/backends/xarray_.py:459]: Replaces the previously single geometry

File: src/geowombat/core/properties.py [hunk 2]
```diff
@@ -884,10 +884,12 @@ def footprint_grid(self) -> gpd.GeoDataFrame:
             )
         import geowombat as gw_
 
+        geometries = []
         if hasattr(self._obj, 'geometries'):
-            geometries = self._obj.geometries
-        else:
-            geometries = []
+            if len(self._obj.geometries) == len(self._obj.gw.filenames):
```
Comment (jgrss) [inline src/geowombat/core/properties.py:889]: Additional check to fix failure during doc plotting

File: src/geowombat/radiometry/mask.py [hunk 3]
```diff
@@ -1,8 +1,10 @@
+import logging
+
 import numpy as np
 import xarray as xr
 
 from ..core import ndarray_to_xarray, norm_diff
-from ..errors import logger
+from ..handler import add_handler
```
Comment (jgrss) [inline src/geowombat/radiometry/mask.py:7]: This is the only relevant change in this file.

File: tests/test_open.py [hunk 4]
```diff
@@ -289,6 +289,37 @@ def test_mosaic_mean(self):
                 )
             )
 
+    def test_footprint_grid(self):
```
Comment (jgrss) [inline tests/test_open.py:292]: New tests

**Comment by jgrss** [comment]
@mmann1123 This PR fixes the failing doc builds. I commented on the relevant changes.

**Review by mmann1123 — APPROVED** [review]

**Pull request merged**